### PR TITLE
README update and file clean-up

### PR DESCRIPTION
**Pull request by MagdalenaZuba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# eZ Platform REST reference
+
+This repository contains the REST API reference for eZ Platform.
+It is build based on RESTful API Modeling Language (RAML), for more information see <https://raml.org/>.
```
Comment (michalmilc90) [inline README.md]:
```suggestion
It is built based on RESTful API Modeling Language (RAML), for more information see <https://raml.org/>.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+The repository is separated into three directories:
+
+- input - RAML definitions and examples of HTTP methods used in eZ Platform
+- output - static HTML automatically generated from RAML definitions
+- raml2html - tool for generating static HTML from RAML definitions
+
+## input
+
+To document new REST API you need to:
```
Comment (michalmilc90) [inline README.md]:
```suggestion
To document new REST API, you need to:
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ 
+- add a new method with errors to `input/ez.raml`
+- add examples of request (e.g. `SectionInput.xml.example`) and response (e.g. `Section.xml.example`) to `input/examples`
+- you should also update descriptions of request and response elements in `input/ez-types.raml`
```
Comment (michalmilc90) [inline README.md]:
```suggestion
3. Update descriptions of request and response elements in `input/ez-types.raml`.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- input - RAML definitions and examples of HTTP methods used in eZ Platform
+- output - static HTML automatically generated from RAML definitions
+- raml2html - tool for generating static HTML from RAML definitions
+
+## input
+
+To document new REST API you need to:
+ 
+- add a new method with errors to `input/ez.raml`
```
Comment (michalmilc90) [inline README.md]:
```suggestion
1. Add a new method with errors to `input/ez.raml`.
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+- add examples of request (e.g. `SectionInput.xml.example`) and response (e.g. `Section.xml.example`) to `input/examples`
```
Comment (michalmilc90) [inline README.md]:
```suggestion
2. Add examples of request (e.g. `SectionInput.xml.example`) and response (e.g. `Section.xml.example`) to `input/examples`.
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+- you should also update descriptions of request and response elements in `input/ez-types.raml`
+
+For detailed information on how to use RAML to document API, see [Specification.](https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/)
+
+## raml2html
+
+To generate static HTML from RAML definitions use the following code:
```
Comment (michalmilc90) [inline README.md]:
```suggestion
To generate static HTML from RAML definitions, use the following code:
```

**Review by michalmilc90 — CHANGES_REQUESTED** [review]

File: raml2html/README.md [hunk 7]
```diff
@@ -5,5 +5,5 @@ Tool for generating static HTML from RAML definitions.
 ## Example usage:
 
 ```sh
-php raml2html.php build --non-standard-http-methods=COPY,MOVE,PUBLISH,SWAP -t default -o examples/ez/output/ examples/ez/ez.raml 
+php raml2html.php build --non-standard-http-methods=COPY,MOVE,PUBLISH,SWAP -t default -o output/ input/ez.raml
```
Comment (adamwojs) [inline raml2html/README.md]: This file could be removed in favor of /README.md 😉

**Review by adamwojs — APPROVED** [review]

**Review by michalmilc90 — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+## input
+
+To document new REST API, you need to:
+ 
+1. Add a new method with errors to `input/ez.raml`.
+2. Add examples of request (e.g. `SectionInput.xml.example`) and response (e.g. `Section.xml.example`) to `input/examples`.
+3. Update descriptions of request and response elements in `input/ez-types.raml`.
+
+For detailed information on how to use RAML to document API, see [Specification.](https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/)
```
Comment (michalmilc90) [inline README.md]:
```suggestion
For detailed information on how to use RAML to document API, see [Specification.](https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md)
```

**Pull request merged**